src/MockBot.Api/Models/Chat.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace MockBot.Api.Models;
+
+public class Chat
+{
+    public Guid Id { get; set; } = Guid.NewGuid();
```

Comment:
Nice! ... this syntax is new in .net 6 and much cleaner than the way we had to set default values before which is detailed in this article: https://www.tutorialsteacher.com/articles/set-default-value-to-property-in-csharp.